.github/workflows/validate.yaml

```diff
@@ -1,33 +1,23 @@
 name: Validate
 on:
-  push:
-    branches-ignore:
+  pull_request:
```

Comment:
This is so strange. From docs I would have never considered `pull_request` as the valid choice for "whenever a push happens to the branch".

The PR details being present simplifies a lot :)